content/docs/gke/gcp-e2e.md

```diff
@@ -803,31 +707,23 @@ Follow these steps to build an image from your code:
 
 Follow these steps to deploy the web UI to your Kubeflow cluster:
 
-1. Move back into your ksonnet application directory:
+The example comes with a simple web front end that can be used with your model.
 
-    ```
-    cd ${WORKING_DIR}/my_ksonnet_app
-    ```
+Enter the `front` directory:
 
-1. Set the ksonnet parameters, using service type 
-   [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) 
-   so that you can connect with your Flask server from outside the cluster:
+```
+cd front
+```
 
-    ```
-    ks param set web-ui image ${UI_IMG_PATH}
-    ks param set web-ui type LoadBalancer
-    ```
+Optionally update the image to the ${UI_IMG_PATH} in the `deployment.yaml`:
 
-1. [Apply][ks-apply] the component to your cluster:
+To deploy the web front end
```

Comment:
Is this phrase part of the previous sentence? It's kind of hanging here without any context and is therefore confusing.

Reply:
Or perhaps there's some code missing here?

Reply:
In fact, from last examples of Kubeflow, there is no way to set image and service type from kustomize. I think we should uodate this from Kubeflow examples repo, and then update here. From current way, that works. see detail from https://github.com/kubeflow/examples/blob/master/mnist/README.md#web-front-end